dev-resources/private/test/src/cljc/foo/bar/core.cljc

```diff
@@ -0,0 +1,16 @@
+(ns foo.bar.core
+  (#?(:clj  :require
+      :cljs :require-macros)
+    [foo.bar.macros]))
```

Comment:
What do you prefer? I vote for my option (of course) but it's a personal preference I guess..